project1/application.py

```diff
@@ -116,3 +119,31 @@ def userHome(Username):
     if Username in session:
         return render_template("userDetails.html", username=Username, message="Successfully logged in.", heading="Welcome back")
     return redirect(url_for('index'))
+<<<<<<< HEAD
+
+@app.route("/search",methods=["POST","GET"])
+def search():
+	if request.method == "GET":
+		return render_template("search.html")
+	else:
+		result = request.form.get("search")
+		result = '%'+result+'%'
+		search_result = Books.query.filter(or_(Books.tittle.ilike(result), Books.author.ilike(result), Books.isbn.ilike(result))).all()
+		return render_template("search.html", books=search_result)
+
+@app.route("/bookpage",methods=["GET"])
+def bookpage():
+	return render_template("bookpage.html")
+||||||| parent of ef5bb0e... last commit for search-resolved conflict
+=======
+
+@app.route("/search",methods=["POST","GET"])
+def search():
+	if request.method == "GET":
+		return render_template("search.html")
+	else:
+		result = request.form.get("search")
+		result = '%'+result+'%'
+		search_result = Books.query.filter(or_(Books.tittle.ilike(result), Books.author.ilike(result), Books.isbn.ilike(result))).all()
```

Comment:
what if there are no search results, how are you handling that?

Reply:
ya im going to add some code to do that, will do that while  integration.